crypto/blake2/blake2b.c

```diff
@@ -14,10 +14,10 @@
  * can be found at https://blake2.net.
  */
 
+#include "e_os.h"
```

Comment:
I thought I tried these without but don't remember the outcome.  Trying again via a commit to see if the AppVeyor builds break.